src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -802,6 +807,13 @@ protected void updateRootTrackers() {
 		}
 	}
 
+	// correct any clipping that is happening to the feet trackers
+	private void tweakLegPos() {
+		// correct the foot positions
+		legTweaks.tweakLegs();
+
```

Comment:
*can you remove the space pwease 3:*

Reply:
yup removing that space and some others I found!